### ISSUE TITLE
Add ability to create detached signatures for files [Enhancement]

### ISSUE THREAD

**Issue by erreip** [gpg4usb/gpg4usb#28]
I would love to have this be my main GPG tool but its missing a critical component (for my purposes): is there a way to add the ability to create a detached signature for any given file?

Cheers,

**Comment by jeruyyap** [comment]
Agreed, it would be great if this could handle detached signatures for files, as that's sometimes a more common use for some GPG users than encrypting and signing messages.

If this was added then gpg4usb would pretty much cover day-to-day GPG use (yeah subkeys and revocation are missing, but those aren't very common tasks).

**Comment by blackhat999** [comment]
This would definitely be a really useful feature.
